code/datums/brain_damage/psychological.dm

```diff
@@ -0,0 +1,301 @@
+/datum/brain_trauma/psychological
```

Comment:
You still havn't adressed my previous points. This is not what brain traumas are. You need to refactor traumas to be more generic and make a psychological and brain subtype.

Reply:
> I made a comment on the exact part thats bad 3 times now.

oh sorry for the longest time my brain couldnt comprehend what i was reading. I think i'm getting what you are saying. So instead of brain_trauma we would have something along the lines of trauma/brain and trauma/psychological right?

Reply:
I made a comment on the exact part thats bad 3 times now.

oh sorry for the longest time my brain couldnt comprehend what i was reading. I think i'm getting what you are saying. So instead of brain_trauma we would have something along the lines of trauma/brain and trauma/psychological right?

Reply:
yes, as right now you're basically working around a lot of the functionality that makes brain traumas brain traumas. instead all the shared behavior should be on datum/trauma and the specific behavior on trauma/brain and trauma/psychological

Reply:
yes now i get it. I was constantly resolving your reviews because my brain just couldnt process "making brain traumas more generic" kek

Reply:
Actually i think i can split it to trauma/brain , trauma/psychological and trauma/event
as there are some traumas that can be aquired only through events

Reply:
depending on what those are those might still fit fine within one of the two previously mentioned categories depending on what they actually do. its about the functionality in the code